From b53451737f4ce4a735876bc56e5a53f04f516c94 Mon Sep 17 00:00:00 2001
From: Tudor Ambarus <tudor.ambarus@microchip.com>
Date: Mon, 7 Oct 2019 11:54:18 +0300
Subject: [PATCH 082/471] crypto: atmel: Add support for the SHA IP available
 on sam9x60

Add version for the SHA IP available on the sam9x60 SoC.

Signed-off-by: Tudor Ambarus <tudor.ambarus@microchip.com>
[ta: ported from 4.19 to 5.4]
Signed-off-by: Tudor Ambarus <tudor.ambarus@microchip.com>
[ta: cherry picked to v5.10]
Signed-off-by: Tudor Ambarus <tudor.ambarus@microchip.com>
[ta: cherry picked to v5.15, drop fall through comment]
Signed-off-by: Tudor Ambarus <tudor.ambarus@microchip.com>
---
 drivers/crypto/atmel-sha.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/crypto/atmel-sha.c b/drivers/crypto/atmel-sha.c
index d1628112dacc..ab192aa2e3f6 100644
--- a/drivers/crypto/atmel-sha.c
+++ b/drivers/crypto/atmel-sha.c
@@ -2509,6 +2509,7 @@ static void atmel_sha_get_cap(struct atmel_sha_dev *dd)
 	/* keep only major version number */
 	switch (dd->hw_version & 0xff0) {
 	case 0x700:
+	case 0x600:
 	case 0x510:
 		dd->caps.has_dma = 1;
 		dd->caps.has_dualbuff = 1;
-- 
2.34.1

